src/index.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import App  from "./components/App/App";
```

Comment:
```suggestion
import App  from "./components/App";
```

при этом нужно создать файл `components/App/index.ts` с примерно таким содержимым:

```ts
export { default } from './App';
```